packages/solid/store/src/store.ts

```diff
@@ -39,21 +39,33 @@ function wrap<T extends StoreNode>(value: T, name?: string): DeepReadonly<T> {
   return p;
 }
 
-export function isWrappable(obj: any) {
-  return (
+function isUnwrappable(obj: {} | undefined | null): obj is NotWrappable {
```

Comment:
This is a type guard, this will work when (if) the TS team decide to improve type guards (these days I also discovered that they don't work in arrays with non-const index).